Update Hydrawise for config flow

dknowles2 opened the pull request:
## Proposed change
Update Hydrawise docs to reflect that it's now set up with config flow.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/95589
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

dknowles2 committed:
Update Hydrawise for config flow

dknowles2 commented:
> It seems that this PR is targeted against an incorrect branch. Documentation updates which apply to our current stable release should target the `current` branch. Please change the target branch of this PR to `current` and rebase if needed. If this is documentation for a new feature, please add a link to that PR in your description.

I think this is wrong? The new feature is based on the core `dev` branch, so updating docs in `current` would be incorrect?

frenck commented on source/_integrations/hydrawise.markdown:
```diff
@@ -26,114 +28,36 @@ There is currently support for the following device types within Home Assistant:
+## Prerequisites
 
-To enable it, add the following to your `configuration.yaml` file:
+In order to set up the Hydrawise integration, you must first obtain an API Key.
```
```suggestion
To set up the Hydrawise integration, you must first obtain an API Key.
```

frenck committed:
Tiny tweak

frenck reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged.

dknowles2 commented:
> > I think this is wrong? The new feature is based on the core `dev` branch, so updating docs in `current` would be incorrect?
> 
> It was correct, you missed adding the link to the Core PR, which you edited afterward:
> 
> ![CleanShot 2023-05-25 at 18 43 44](https://user-images.githubusercontent.com/195327/240980656-86fd3086-dc95-482c-ab75-1aa34481ddf1.png)
> 
> Because the parent PR was missing, it assumed it wasn't related to a code change (and thus targeting an incorrect branch).
> 
> That said, all is good now 👍
> 
> ../Frenck

Ah, that makes sense! I wasn't sure which one to submit first since both PR templates request a reference to the other. 😵

frenck committed:
2023.7: Beta release notes (#27723)

frenck committed:
2023.7: Revert some breaking changes, small tweak

dannytsang committed:
Updated instructions to create a local user for Unifi Protect Integration (#28003)

* Updated instructions to create a local user.

Based on Unifi OS 3.0.20.

* Fixed capitalisation of "IP".

* Update source/_integrations/unifiprotect.markdown

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

jbouwh committed:
Fix incorrect type and default url_topic option mqtt image (#28015)

dknowles2 committed:
Merge branch 'next' into hydrawise

dknowles2 committed:
Remove stray keystrokes

marcelveldt committed:
Small improvements to the Matter documentation (#28010)

* add link to thread/docker documentation

* remove redundant text about current state

platform support is already mentioned in the top, info is outdated

* remove redundant or outdated info

* add text about running matter server in docker

* add switchbot docs

* another note about thread

* Update source/_integrations/matter.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/matter.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/matter.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/matter.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/matter.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/matter.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/matter.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/matter.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/matter.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/matter.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Fix terminology issues I introduced with my review comments

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

EnthusiastDeveloper committed:
Fix typo ibeacon.markdown (#28024)

Replace "light of sight" with the correct English term

APayerl committed:
Update modbus.markdown (#28028)

Add link to light section in modbus documentation

grapesmith committed:
Add unsupported model section for TP-Link Kasa integration (#28042)

* Add unsupported model section for TP-Link Kasa integration

Ref: https://github.com/home-assistant/core/issues/94947

Some newer models of TP-Link Kasa devices are incompatible with the integration. Creating a section for them and adding one known model. Additionally, cross link to the Matter integration where it has limited support.

* tiny tweak

change link to point to link to the topic title within the matter page

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

dependabot[bot] committed:
Bump textlint from 13.3.2 to 13.3.3 (#28044)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

marcelveldt committed:
Add climate platform to Matter docs (#28009)

* Add climate platform to Matter docs

* adjust text about missing thermostat support on aqara hub

* adjust category

piitaya committed:
Adds Assist action documentation (#28047)

* Adds assist action documentation

* Apply suggestions

* Fix other actions

trickykid1982 committed:
DenonAvr add AVR-1713 as supported (#28049)

ultrafez committed:
Make clearer that all automation conditions must pass (#28052)

* Make clearer that all automation conditions must pass

Clarify that if any automation conditions fail, the automation actions will not be executed.

* tiny tweaks

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

emontnemery committed:
Remove documentation of deprecated vacuum services (#28054)

mehuman committed:
Troubleshooting section no longer needed after move to new library (#28053)

emontnemery committed:
Remove documentation of deprecated MQTT vacuum protocol (#28056)

emontnemery committed:
Add documentation of deprecated vacuum services to ecovacs (#28055)

starkillerOG committed:
Add Brel HUB-3 as supported (#28059)

c0ffeeca7 committed:
Explanation of conditions: Rephrase trigger statement (#28057)

Petro31 committed:
Simple fix to clarify information. (#28065)

Co-authored-by: gleversluis <46139776+gleversluis@users.noreply.github.com>

c0ffeeca7 committed:
Common tasks: add enable entities (#28046)

jbouwh committed:
Alternative for updating discovery examples (#28026)

* Alternative for updating discovery examples

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

mehuman committed:
Setting fan speed is also supported (#28069)

enygren committed:
Update purpleair.markdown to include instructions to get an API key (#27852)

* Update purpleair.markdown to include instructions to get an API key

Add instructions on how to create a Purple Air API Key needed to enable the integration.  Addresses ttps://github.com/home-assistant/core/issues/93663

* Apply suggestions from code review

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

tkdrob committed:
Clarify IMAP character set (#27163)

* Clarify IMAP character set

* typo

* Apply suggestions from code review

* Update source/_integrations/imap.markdown

* Tiny tweak

* Fix parser context switching

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>
Co-authored-by: Franck Nijhof <frenck@frenck.nl>

c0ffeeca7 committed:
Installation Mac-OS: remove KVM instructions (#27515)

TheGabeMan committed:
Update roborock.markdown (#27905)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

sean797 committed:
virt-install requires `--import` (#28067)

qcow2 image require `--import`

frenck committed:
Merge branch 'current' into rc

maximoei committed:
Update telegram_bot docs to reference use of URLs in InlineKeyboards (#22599)

akx committed:
sensirion-ble: note SHT4x and SHT3x are supported too in 2023.7 (#28073)

c0ffeeca7 committed:
Glossary: fix broken link (#28075)

Mettchen committed:
Update buienradar.markdown (#27984)

* Update buienradar.markdown

Changed wording of camera entity sentence. Added information on how to enable the camera entity.

* add reference to procedure on enabling entities

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

frenck committed:
2023.7: Finish breaking changes

frenck committed:
2023.7: Update changelog

frenck committed:
2023.7: Sync docs with codebase

frenck committed:
2023.7: Finish new lock entity dialog section

frenck committed:
2023.7: Finish new script dialog

frenck committed:
2023.7: Adjust title, date & description

frenck committed:
2023.7: Write introduction

frenck committed:
2023.7: Finish sentence trigger

frenck committed:
2023.7: Finish image entity section

frenck committed:
2023.7: Finish reduce breaking change risk

frenck committed:
2023.7: Finish iteration of the integrations dashboard

frenck committed:
2023.7: Finish timezone selection section

frenck committed:
2023.7: Finish new integrations section

frenck committed:
2023.7: Finish unavailable automations section

frenck committed:
2023.7: Re-order sections

frenck committed:
2023.7: Finish other noteworthy section

frenck committed:
2023.7: Finish matter updates section

frenck committed:
2023.7: Finish open assist action

frenck committed:
2023.7: Finish copy n paste support for dashboards

bramkragten committed:
Add conversation agent selector documentation (#28077)

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

frenck committed:
2023.7: Finish bluetooth

frenck committed:
2023.7: Finish self-sufficient addition

frenck committed:
2023.7: Finish self-sufficient addition tweaks

frenck committed:
2023.7: Finish service responses

frenck committed:
2023.7: Process review comments

frenck committed:
Merge branch 'current' into rc

frenck committed:
2023.7: Process review comments

frenck committed:
2023.7: Process review comments

bdraco committed:
Add slow connection times troubleshooting section to SwitchBot (#28083)

* Add troubleshooting section to SwitchBot

* rework

dependabot[bot] committed:
Bump actions/setup-node from 3.6.0 to 3.7.0 (#28089)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump nokogiri from 1.15.2 to 1.15.3 (#28088)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

c0ffeeca7 committed:
Integrations page update paths to reflect current UI (#28091)

* Integrations page: update paths to reflect current UI

- The cogwheel on the integration card was removed with 2023.7

* Undo automatic numbering

jbouwh committed:
Fix mqtt discovery redirect (#28090)

c0ffeeca7 committed:
Getting started: update screenshot (#28099)

balloob committed:
Promo year of the voice chapter 3 (#28104)

balloob committed:
2023.7.1 (#28100)

dknowles2 committed:
Merge branch 'home-assistant:current' into hydrawise
